e2e/test/provisioning/ProvisioningE2ETests.cs

```diff
@@ -1428,6 +1428,23 @@ public static ProvisioningClientOptions CreateProvisioningClientOptionsFromName(
             return auth;
         }
 
+        private string GetHostName(string connectionString)
```

Comment:
Since this logic is being used in more than 1 place, you could add this into a helper class